### PR TITLE
Added unit tests for media element

### LINKED ISSUE #3185: Media dropdown menu not shown for videos
## Bug Description

We display these three dots when hovering over media items, which allows opening a menu to edit & delete media.

Apparently, this menu is currently missing for videos.

## Expected Behaviour

I should be able to edit and delete video items.

## Steps to Reproduce

1. Hover over video.
1. See no three-dots-button

## Screenshots

Here's what it looks like for images:

Screenshot 2020-07-14 at 11 13 17

For videos, the button is simply missing.

## Additional Context

- Plugin Version: `1.0.0-beta.1`
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by joannag6**
## Summary

Added unit tests for media element. This is just scoped to testing the fix for the bug in #3185

## User-facing changes

Added some aria labels as well.

---

Fixes #3185

### CODE REVIEW

File: assets/src/edit-story/components/library/panes/media/common/mediaElement.js [hunk 1]
```diff
@@ -209,7 +209,11 @@ const MediaElement = ({
           setShowVideoDetail(false);
           if (mediaElement.current) {
             // Pointer still in the media element, continue the video.
-            mediaElement.current.play().catch(() => {});
+            const playPromise = mediaElement.current.play();
+            if (playPromise) {
```
Comment (joannag6) [inline assets/src/edit-story/components/library/panes/media/common/mediaElement.js:213]: Reverted change from https://github.com/google/web-stories-wp/pull/2286/commits/efb575a3d5bbaf107f59ae6ccc4b826915d62141

I think the unit test runner doesn't return a promise so the test fails here.

File: assets/src/edit-story/components/library/test/mediaElement.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+/**
+ * Internal dependencies
+ */
+import { Simulate } from 'react-dom/test-utils';
+import MediaElement from '../panes/media/common/mediaElement';
+import { ProviderType } from '../panes/media/common/providerType';
+import { renderWithTheme } from '../../../testUtils';
+
+function renderMediaElement(resource, providerType) {
+  const { getByAriaLabel, queryByAriaLabel } = renderWithTheme(
```
Comment (diegovar) [inline assets/src/edit-story/components/library/test/mediaElement.js]: Why not just return the result of renderWithTheme?

File: assets/src/edit-story/components/library/test/mediaElement.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+      local: false, // Already uploaded
+      alt: 'image :)',
+    };
+
+    const { getByAriaLabel } = renderMediaElement(resource, ProviderType.LOCAL);
+
+    const element = getByAriaLabel('image :)');
+    Simulate.focus(element);
+
+    expect(getByAriaLabel('More')).toBeInTheDocument();
```
Comment (diegovar) [inline assets/src/edit-story/components/library/test/mediaElement.js:62]: Can you assert that it was not in the document before focusing it?

**Review by diegovar — APPROVED** [review]

File: assets/src/edit-story/components/library/test/mediaElement.js (hunk 2 above)
Comment (joannag6) [reply assets/src/edit-story/components/library/test/mediaElement.js]: Done.

File: assets/src/edit-story/components/library/test/mediaElement.js (hunk 3 above)
Comment (joannag6) [reply assets/src/edit-story/components/library/test/mediaElement.js:62]: Done.

**Pull request merged**